ulib/ulibfs.fsproj

```diff
@@ -85,10 +84,22 @@
     <Compile Include="fs\FStar_Mul.fs" />
     <Compile Include="fs\FStar_Option.fs" />
     <Compile Include="fs\FStar_Pervasives_Native.fs" />
-    <Compile Include="fs\FStar_Set.fs" />
+    <!-- TODO: Set is implemented with extra 'comparison' constraint on key. 
```

Comment:
I suppose this is also a problem with OCaml implementation, as BatSet uses 'compare' function internally for ordering elements?